test/integration/clusterquota_test.go

```diff
@@ -0,0 +1,168 @@
+// +build integration
+
+package integration
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	testutil "github.com/openshift/origin/test/util"
+	testserver "github.com/openshift/origin/test/util/server"
+)
+
+const (
+	numProjects             = 100
+	podsPerProjectPerSecond = 20.0
+	testLength              = 30 * time.Second
+)
+
+func TestClusterQuota(t *testing.T) {
```

Comment:
@derekwaynecarr  This test is supposed to create 100 projects and start making requests in each project.  All the projects are under a single cluster quota doc (so they all contend for the lock).  

When I run it each project is able to request 2.6 resources per second waiting for a response before proceeding.  That last bit matters because you can push even more through if you request concurrently like our infrastructure actually does.  I'm easily able to get double that.  Beyond that I'm hitting open file limits on my machine and don't feel like rebooting.

Can you double check the test for bugs?  I had to switch from pods to configmaps because the scheduler started having trouble and was eating my rate limits.